cluster/gce/gci/configure-helper.sh

```diff
@@ -576,7 +604,9 @@ function start-kubelet {
     fi
   else # For nodes
     flags+=" --enable-debugging-handlers=true"
-    flags+=" --api-servers=https://${KUBERNETES_MASTER_NAME}"
+    flags+=" --experimental-bootstrap-kubeconfig=/etc/kubernetes/kube-csr.json"
+    flags+=" --require-kubeconfig"
+    flags+=" --kubeconfig=/etc/kubernetes/kubelet-kubeconfig.json"
```

Comment:
not .json? this isn't an existing file, right? it's the one expected to be populated by bootstrap?